example-apps/react-spa-local-first/src/App.tsx

```
@@ -0,0 +1,974 @@
+      }
+    };
+
+    for (const endpoint of endpoints) {
+      const controller = new AbortController();
+      controllers.set(endpoint.id, controller);
+      void fetchDescribe(endpoint, controller.signal);
+    }
+
+    return () => {
+      cancelled = true;
+      controllers.forEach((controller) => controller.abort());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)